From c9b3cfcab30ace06c21f9b48a0f0a34805e81e9d Mon Sep 17 00:00:00 2001
From: V8 Autoroll <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Tue, 27 Oct 2026 07:07:21 -0700
Subject: [PATCH] Version 15.6.75.27

Version incremented at https://cr-buildbucket.appspot.com/build/8666781798487798769

Change-Id: Iaaa92660cb31bfcb5507d9075dc82ca2634b3a2d
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/8668384
Bot-Commit: v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/15.6@{#54}
Cr-Branched-From: b976f35da8356fcf4e5e176d1d810ba6a6f05718-refs/heads/15.6.75@{#1}
Cr-Branched-From: a424b0597aab8e27f12ea4a700bae3961fa1a076-refs/heads/main@{#110135}
---

--- a/v8/include/v8-version.h
+++ b/v8/include/v8-version.h
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 15
 #define V8_MINOR_VERSION 6
 #define V8_BUILD_NUMBER 75
-#define V8_PATCH_LEVEL 25
+#define V8_PATCH_LEVEL 27
 
 // Use 1 for candidates and 0 otherwise.
 // (Boolean macro values are not supported by all preprocessors.)
